po/nicotine.pot

```diff
@@ -33,6 +33,10 @@ msgstr ""
 msgid "Graphical client for the Soulseek peer-to-peer network"
 msgstr ""
 
+#: data/org.nicotine_plus.Nicotine.desktop.in:8
+msgid "org.nicotine_plus.Nicotine"
```

Comment:
The icon name shouldn't be here, but I'm not sure what caused this.

Reply:
Probably this: https://savannah.gnu.org/support/?108887

Reply:
Thanks! Upgraded to 0.21 now and it's gone.